static/styles/app_components.css

```diff
@@ -563,6 +564,10 @@ div.overlay {
     color: hsl(0, 0%, 100%);
 }
 
+.unread_mention_info {
+    margin-right: 2px;
+}
```

Comment:
Should this CSS rule apply to `:not(:empty)`, or do you want it to take up space even when empty?

Reply:
Hmmm, Sure!
Thanks for this!